{
	"vulinfoid": "JVNDB-2024-001553",
	"vulinfodata": {
		"title": "GNU Project の GnuTLS におけるデジタル署名の検証に関する脆弱性",
		"vulinfodescription": {
			"overview": "GNU Project の GnuTLS には、デジタル署名の検証に関する脆弱性が存在します。"
		},
		"affected": {
			"affecteditem": [
				{
					"name": "GNU Project",
					"productname": "GnuTLS",
					"cpe": {
						"text": "cpe:/a:gnu:gnutls",
						"version": "2.2"
					},
					"versionnumber": [
						"3.8.3 未満"
					]
				}
			]
		},
		"impact": {
			"cvss": [
				{
					"version": "3.0",
					"severity": {
						"text": "High",
						"type": "Base"
					},
					"base": "7.5",
					"vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
				}
			],
			"impactitem": {
				"description": "サービス運用妨害 (DoS) 状態にされる可能性があります。"
			}
		},
		"solution": {
			"solutionitem": {
				"description": "ベンダアドバイザリまたはパッチ情報が公開されています。参考情報を参照して適切な対策を実施してください。"
			}
		},
		"related": {
			"relateditem": [
				{
					"type": "advisory",
					"name": "Common Vulnerabilities and Exposures (CVE)",
					"vulinfoid": "CVE-2024-0567",
					"url": "https://www.cve.org/CVERecord?id=CVE-2024-0567"
				},
				{
					"type": "advisory",
					"name": "National Vulnerability Database (NVD)",
					"vulinfoid": "CVE-2024-0567",
					"url": "https://nvd.nist.gov/vuln/detail/CVE-2024-0567"
				},
				{
					"type": "advisory",
					"name": "関連文書",
					"vulinfoid": "gitlab.com (issues/1521)",
					"url": "https://gitlab.com/gnutls/gnutls/-/issues/1521"
				},
				{
					"type": "advisory",
					"name": "関連文書",
					"vulinfoid": "lists.gnupg.org (004841)",
					"url": "https://lists.gnupg.org/pipermail/gnutls-help/2024-January/004841.html"
				},
				{
					"type": "advisory",
					"name": "関連文書",
					"vulinfoid": "www.openwall.com (oss-security/2024/01/19/3)",
					"url": "http://www.openwall.com/lists/oss-security/2024/01/19/3"
				},
				{
					"type": "advisory",
					"name": "関連文書",
					"vulinfoid": "access.redhat.com (CVE-2024-0567)",
					"url": "https://access.redhat.com/security/cve/CVE-2024-0567"
				},
				{
					"type": "advisory",
					"name": "関連文書",
					"vulinfoid": "bugzilla.redhat.com (2258544)",
					"url": "https://bugzilla.redhat.com/show_bug.cgi?id=2258544"
				},
				{
					"type": "cwe",
					"name": "JVNDB",
					"vulinfoid": "CWE-347",
					"url": "https://cwe.mitre.org/data/definitions/347.html",
					"title": "デジタル署名の不適切な検証"
				}
			]
		},
		"history": {
			"historyitem": [
				{
					"historyno": "1",
					"datetime": "2024-02-05T15:42:50+09:00",
					"description": "[2024年02月05日]\\n  掲載"
				}
			]
		},
		"datefirstpublished": "2024-02-05T15:42:50+09:00",
		"datelastupdated": "2024-02-05T15:42:50+09:00",
		"datepublic": "2024-01-16T00:00:00+09:00"
	}
}
